src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,27 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(ActorDao actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        if (actorDao.get(id).isEmpty()) {
+            throw new DataProcessingException("Can't find actor by id: " + id);
+        }
+        return actorDao.get(id).get();
```

Comment:
1. Use orElseThrow method to make code shorter
2. DataProcessingException is used on dao layer to indicate problems in communication with DB (situation where result cannot be determined). In service another type of exception should be used as operation with DB was successful and we can properly reply that we don't have requested data
```suggestion
        return actorDao.get(id)
                .orElseThrow(() -> new EntityNotFoundException("Can't find actor by id: " + id));
```